libbeat/processors/add_kubernetes_metadata/config.go

```diff
@@ -43,11 +47,33 @@ type Enabled struct {
 
 type PluginConfig []map[string]common.Config
 
+func getSystemKubeConfig() string {
+	envKubeConfig := os.Getenv("KUBECONFIG")
+	if _, err := os.Stat(envKubeConfig); !os.IsNotExist(err) {
+		return envKubeConfig
+	}
+	homeKubeConfig := os.Getenv("HOME") + "/.kube/config"
```

Comment:
What about using 
`filepath.Join(os.Getenv("HOME"), ".kube", "config")`
to support other OSes?